src/fake-timers-src.js

```diff
@@ -1222,7 +1233,7 @@ function withGlobal(_global) {
             const result = addTimer(clock, {
                 func: func,
                 delay: getTimeToNextFrame(),
-                args: [clock.now + getTimeToNextFrame()],
+                args: [FakePerformanceNow],
```

Comment:
This is where I would create the closure, if possible.

Reply:
Got it! I think it's better but let me know what you think!